devnotes/mvpd_generation.md

```diff
@@ -0,0 +1,20 @@
+# MVPD data modification
+
+## Testing environement
+
+Original `PNOR` image was aquired by building `op-build`.
```

Comment:
please link to relevant documentation if we have one

Reply:
Added a link to a doc, how to build `op-build`